src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -212,7 +212,7 @@ public string Version
         /// <summary>
         /// Name of the Host (i.e. DynamoRevit/DynamoStudio)
         /// </summary>
-        [Obsolete("This property will be removed in Dynamo 3.0 - please use HostAnalyticsInfo")]
+        [Obsolete("This property will be removed in a future version of Dynamo - please use HostAnalyticsInfo")]
         public string HostName { get; set; }
```

Comment:
too complicated ?
Could we make these internal at least ?

Reply:
Good point. I will make a pass.